walkthroughs/step2.md

```diff
@@ -1,34 +1,35 @@
-Let's go ahead and create an application. We will be using Sinatra as a framework for our application. Thet means that we will be using Ruby as a
-programming language. You can read more about Sinatra at....
-
+Let's go ahead and create an application. We will be using Sinatra as a framework for our application. That means that we will be using Ruby as a
+programming language. You can read more about [Sinatra here](http://www.sinatrarb.com/)
 
 There are several ways to create a new Sinatra application but we will make use of a
 script that creates an app for us AND configures it (at least partially) for use with Cucumber and another testing framework called RSpec (more on that later.
 
-In you terminal, install the following gem:
-
+Open your `Gemfile` and add the following lines:
 ```
-gem install cucumber-sinatra
+gem 'sinatra'
+gem 'cucumber-sinatra'
 ```
 
+Then run `bundle install` to install the new gem we just added.
+
 after the installation is complete, run:
 
 ```
 cucumber-sinatra init --app WorkshopApp application.rb
 ```
 
-There wil be a lot of new files created and we will go over all of them in a while. For now, I want you to
-open up your `Gemfile` and add:
+There will be a lot of new files created and we will go over all of them in a while. For now, I want you to open up your `Gemfile` again and add:
 
 ```
 gem 'rspec'
```

Comment:
@tochman doesn't look like we need `rspec` just yet at this stage, how about we move this to where we actually start using it